start.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
```

Comment:
Sneaking this into this PR haha, because it bugs me every time I accidentally write `./start.sh` and get `./start.sh: 16: Syntax error: "(" unexpected`. `sandbox.sh` is already using `/bin/bash` so at the very least we should be consistent

Reply:
It turns out that on arch, `sh` is just a link to `bash`, so that's why this has been working for me and Edu but not you. Good catch.